mmdet/models/detectors/two_stage.py

```diff
@@ -40,8 +42,12 @@ def __init__(self,
             self.bbox_head = builder.build_head(bbox_head)
 
         if mask_head is not None:
-            self.mask_roi_extractor = builder.build_roi_extractor(
-                mask_roi_extractor)
+            if mask_roi_extractor is not None:
+                self.mask_roi_extractor = builder.build_roi_extractor(
+                    mask_roi_extractor)
+                self.shared_roi_extractor = False
+            else:
+                self.shared_roi_extractor = True
```

Comment:
`self.mask_roi_extractor = self.bbox_roi_extractor` can be added here, which will make the inference more simple.